File: AppGenerateFiles/dbo/Model/Transactional_Valoracion.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transactional_Valoracion : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_valoracion { get; set; }
       public string? Descripcion { get; set; }
       public string? Marca { get; set; }
       public string? Modelo { get; set; }
       public Double? Tasa_interes { get; set; }
       public int? Plazo { get; set; }
       public DateTime? Fecha { get; set; }
       public Double? Tasa_de_cambio { get; set; }
       public int? id_estado { get; set; }
       public Double? Valoracion_compra_cordobas { get; set; }
       public Double? Valoracion_compra_dolares { get; set; }
       public Double? Valoracion_empeño_cordobas { get; set; }
       public Double? Valoracion_empeño_dolares { get; set; }
       public string? Serie { get; set; }
       public int? id_categoria { get; set; }
       public Double? Precio_venta_empeño_dolares { get; set; }
       public Double? Precio_venta_empeño_cordobas { get; set; }
       [ManyToOne(TableName = "Catalogo_Categoria", KeyColumn = "id_categoria", ForeignKeyColumn = "id_categoria")]
       public Catalogo_Categoria? Catalogo_Categoria { get; set; }
       [ManyToOne(TableName = "Catalogo_Estados_Articulos", KeyColumn = "id_estado_articulo", ForeignKeyColumn = "id_estado")]
       public Catalogo_Estados_Articulos? Catalogo_Estados_Articulos { get; set; }
       [OneToMany(TableName = "Detail_Prendas", KeyColumn = "id_valoracion", ForeignKeyColumn = "id_valoracion")]
       public List<Detail_Prendas>? Detail_Prendas { get; set; }
       [OneToOne(TableName = "Detail_Valores", KeyColumn = "id_valoracion", ForeignKeyColumn = "id_valoracion")]
       public Detail_Valores? Detail_Valores { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transactional_Valoracion.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Categoria }  from './Catalogo_Categoria.js'
import { Catalogo_Estados_Articulos }  from './Catalogo_Estados_Articulos.js'
import { Detail_Prendas }  from './Detail_Prendas.js'
import { Detail_Valores }  from './Detail_Valores.js'
class Transactional_Valoracion extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_valoracion;
   /**@type {String}*/ Descripcion;
   /**@type {String}*/ Marca;
   /**@type {String}*/ Modelo;
   /**@type {Number}*/ Tasa_interes;
   /**@type {Number}*/ Plazo;
   /**@type {Date}*/ Fecha;
   /**@type {Number}*/ Tasa_de_cambio;
   /**@type {Number}*/ Valoracion_compra_cordobas;
   /**@type {Number}*/ Valoracion_compra_dolares;
   /**@type {Number}*/ Valoracion_empeño_cordobas;
   /**@type {Number}*/ Valoracion_empeño_dolares;
   /**@type {String}*/ Serie;
   /**@type {Number}*/ Precio_venta_empeño_dolares;
   /**@type {Number}*/ Precio_venta_empeño_cordobas;
   /**@type {Catalogo_Categoria} ManyToOne*/ Catalogo_Categoria;
   /**@type {Catalogo_Estados_Articulos} ManyToOne*/ Catalogo_Estados_Articulos;
   /**@type {Array<Detail_Prendas>} OneToMany*/ Detail_Prendas;
   /**@type {Array<Detail_Valores>} OneToMany*/ Detail_Valores;
}
export { Transactional_Valoracion }
